### DISCUSSION TITLE
Changes to the limit-request-body parameter did not effect

### DISCUSSION THREAD

**Discussion by wesleynf** [cvat-ai/cvat#2791] [General]
Hello,
   I need to upload large images ( about 2GB )  to new project, I've changed the parameter limit-request-body on file supervisord.conf after that I stopped and start CVAT using docker-compose command.
   
   My problem is that the new value isn't load, the CVAT server is using the original parameter,  What should I do ?
   
Show config file bellow:

command=%(ENV_HOME)s/wait-for-it.sh %(ENV_CVAT_POSTGRES_HOST)s:5432 -t 0 -- bash -ic \
    "rm -f /tmp/cvat-server/httpd.pid && python3 ~/manage.py migrate && \
    python3 ~/manage.py collectstatic --no-input && \
    exec python3 $HOME/manage.py runmodwsgi --log-to-terminal --port 8080 \
    --limit-request-body **107374182400** --include-file ~/mod_wsgi.conf \
    %(ENV_DJANGO_MODWSGI_EXTRA_ARGS)s --locale %(ENV_LC_ALL)s \
    --server-root /tmp/cvat-server"

Show logs bellow:

2021-02-10 14:39:42,633 DEBG 'runserver' stderr output:
[Wed Feb 10 14:39:42.633349 2021] [http:info] [pid 306:tid 139878098269952] [client 172.28.0.6:33288] AH01588: Requested content-length of 1586950677 is larger than the configured limit of 1073741824, referer: http://localhost:8080/tasks/create?projectId=2

**Comment by azhavoro** [comment]
@wesleynf Hi, you need to rebuild images with `docker-compose up -d --build`, also this settings should be adjusted as well https://github.com/openvinotoolkit/cvat/blob/develop/cvat/settings/base.py#L426

**Reply by wesleynf** [reply]
It's working now !!!! Thanks a lot @azhavoro

**Reply by Pawel842** [reply]
Hi,
I have changed the `supervisord.conf`   `--limit-request-body to -> 10737418240`
and edited:
https://github.com/openvinotoolkit/cvat/blob/develop/cvat/settings/base.py#L426 to ->
```
DATA_UPLOAD_MAX_MEMORY_SIZE = 1000 * 1024 * 1024  # 1000 MB
DATA_UPLOAD_MAX_NUMBER_FIELDS = None   # this django check disabled
LOCAL_LOAD_MAX_FILES_COUNT = 5000
LOCAL_LOAD_MAX_FILES_SIZE = 5120 * 1024 * 1024  # 5120 MB
```
and then run:
`docker-compose up -d --build`
next even:
`docker-compose up --build --force-recreate`

but the error  `Requested content-length of 1411220677 is larger than the configured limit of 1073741824` still occurs.
Do you have any idea what else can be done?

**Reply by azhavoro** [reply]
> Hi, I have changed the `supervisord.conf` `--limit-request-body to -> 10737418240` and edited: https://github.com/openvinotoolkit/cvat/blob/develop/cvat/settings/base.py#L426 to ->
> 
> ```
> DATA_UPLOAD_MAX_MEMORY_SIZE = 1000 * 1024 * 1024  # 1000 MB
> DATA_UPLOAD_MAX_NUMBER_FIELDS = None   # this django check disabled
> LOCAL_LOAD_MAX_FILES_COUNT = 5000
> LOCAL_LOAD_MAX_FILES_SIZE = 5120 * 1024 * 1024  # 5120 MB
> ```
> 
> and then run: `docker-compose up -d --build` next even: `docker-compose up --build --force-recreate`
> 
> but the error `Requested content-length of 1411220677 is larger than the configured limit of 1073741824` still occurs. Do you have any idea what else can be done?

Hi, you actually don't build any images which docker-compose should notify about, try `docker-compose -f docker-compose.yml -f docker-compose.dev.yml build`

**Reply by giuseta** [reply]
Hi have the same issue when trying to upload a project backup
